programmers/난이도별/level01.2016년/HyeonJeong.py

```diff
@@ -1,44 +1,22 @@
 def solution(a, b):
-    answer = ''
-    day_list = ["MON", "TUE", "WED", "THU", "FRI", "SAT", "SUN"]
-    i = b % 7
 
-    if ( a == 1 or a == 4 or a == 7):
-        if(4 <=i <= 6):
-            answer = day_list [i-4]
-        else:
-            answer = day_list [i+3]
+    total = b
+    day_list = ["THU", "FRI", "SAT", "SUN", "MON", "TUE", "WED"]
 
-    elif (a == 2 or a == 8):
-        if(i == 0):
-            answer = day_list [6]
-        else:
-            answer = day_list [i-1]
+    months_31 = [1, 3, 5, 7, 8, 10, 12]
+    months_30 = [4, 6, 9, 11]
+    months_29 = [2]
 
-    elif (a == 3 or a == 11):
-            answer = day_list [i]
+    for i in range (1, a):
+        if i in months_31:
+            total += 31
+        elif i in months_30:
+            total += 30
+        elif i in months_29:
+            total += 29
```

Comment:
```suggestion
        else:
            total += 29
```
네 이렇게 해도 상관없습니다!

그리고 if, else, else if 등을 너무 많이 사용하면 문제에 오류가 생긴다기 보단, 코드를 읽을 때 다른 사람들이 이해하기가 어려울 수 있어요! 나중엔 자기가 작성한 코드인데도 무슨 소리인가 싶을 때가 많습니다.

개발은 혼자서 하는게 아니라 다른 사람들과 같이 하는 것이기 때문에 항상 다른 사람들도 읽기 쉬운 코드를 작성해야 합니다!

Reply:
혼자 해볼때는 몰랐는데, 읽기 쉬운 코드가 정말 중요하네요!!

Reply:
혼자 할 때는 몰랐는데, 다른 사람들도 읽기 쉽게 만드는 것이 정말 중요한 것을 다시 알게되네요!!